packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -205,7 +206,11 @@ export default class VList {
             }
 
             item.writeBack(listStack, this.rootList, shouldReuseAllAncestorListElements);
-            const topList = listStack[1];
+            const topList = listStack[1] as HTMLElement;
+            if (topList) {
+                topList.style.marginBlockStart = '0px';
```

Comment:
What about an existing list that has margins before and after? Will this cause the margins go away suddenly?